docs/Tutorial-3_Extracting_Data_and_Trajectory_Analysis.md

```diff
@@ -15,13 +15,11 @@ conda install "ipywidgets <8" -c conda-forge
 ```
 
 !!! note "NOTE"
-
-        You may have to restart vscode or other editor after updating `ipywidgets`.
+    You may have to restart your IDE after updating `ipywidgets`.
```

Comment:
I don't think this is a term that everyone necessarily knows. I would rather use a less technical term or add a clarification like "i.e., your code editor" or something like that

Reply:
Okay, I'll put "code editor (IDE)"

Reply:
great :+1: